### ISSUE TITLE
'confirmationRequired' optional param is not available in latest ionic-native/fingerprint-aio package

### ISSUE THREAD

**Issue by Seabird22** [NiklasMerz/cordova-plugin-fingerprint-aio#333]
## Description

* What did you do?
* What did happen?
* What did you expect to happen?

## Environment

* Plugin version
* Build environment (```cordova info```)
* ```cordova platform ls```

### Logs

<details>
  <pre>[Please insert any logs here]</pre>
</details>

**Comment by dmitry-salnikov** [comment]
Yes, it would be nice to update ionic wrapper, the `allowBackup` parameter is missing ther too. #351
